docs/api/command-line-switches.md

```diff
@@ -321,3 +321,16 @@ The default is `verbatim` and `dns.setDefaultResultOrder()` have higher priority
 [play-silent-audio]: https://github.com/atom/atom/pull/9485/files
 [ready]: app.md#event-ready
 [severities]: https://source.chromium.org/chromium/chromium/src/+/main:base/logging.h?q=logging::LogSeverity&ss=chromium
+
+
+## Chromium Flags
+
+While a list of all available Chromium switches doesn't exist, there are a few ways of discovering them.
+
+The easiest method is Chromium's flags, via `about://flags`, while those flags don't translate 1:1 to their switch names, they are listed in the process's Command Line argument.
+
+You can enable a flag, then navigate to `about://version` in your Chromium browser, where you can find the list of Command Line arguments. There you can find `--flag-switches-begin --your --list --of --switches --flag-switches-end`.
+
+While most flags are an argument to `--enable-features=`, some are standalone, such as `--enable-experimental-web-platform-features`. Do note that many switches can be split up into individual features, so this list is non-exhaustive.
```

Comment:
The Chromium developer docs lists https://peter.sh/experiments/chromium-command-line-switches/ as a non-exhaustive list to reference.

Reply:
fixed